chromium_src/chrome/common/importer/profile_import_process_param_traits_macros.h

```diff
@@ -10,9 +10,9 @@
 #include "chrome/common/importer/importer_type.h"
 
 #if !BUILDFLAG(IS_WIN)
-#define TYPE_BOOKMARKS_FILE TYPE_EDGE_CHROMIUM
+#define TYPE_BOOKMARKS_FILE TYPE_VIVALDI
```

Comment:
I think we can remove the `#if !BUILDFLAG(IS_WIN)` guard here since Vivaldi is on both platforms (same with `#undef` below)

Reply:
yep, done

Reply:
removed

Reply:
pushed it back, it still needed becasue otherwise compilation failed when building from scratch. We added new values with ids more than last ones mentioned in Chromium code. cc @bsclifton

Reply:
ah thanks! I think we can still unify the logic and put after bookmarks file. For example, removing the guards like `#if !BUILDFLAG(IS_WIN)`